testing/xvkbd/APKBUILD

```diff
@@ -0,0 +1,43 @@
+# Contributor: Jan Hustak <coding@journey.sk>
+# Maintainer:
+pkgname=xvkbd
+pkgver=3.9
+pkgrel=0
+pkgdesc="The xvkbd virtual keyboard for X11"
+url="http://t-sato.in.coocan.jp/xvkbd/"
+arch="all"
+license="GPL-2.0-or-later"
+options="!check" # No test suite
+depends="libxt libxmu libxaw libxtst"
+depends_dev="imake xorg-server-dev libxt-dev libxmu-dev
```

Comment:
`depends_dev` without a `-dev` package does not make a lot of sense. Just add these dependenciesto `makedepends` directly.